components/admin/AdminTable/index.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Button,
+  ButtonGroup,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+} from '@material-ui/core';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import InfoIcon from '@material-ui/icons/Info';
+import styles from './AdminTable.module.css';
+
+const declineButton = (
+  <Button variant="outlined" size="small">
+    Decline
+  </Button>
+);
+
+const deleteButton = (
+  <Button variant="outlined" size="small">
+    Delete
+  </Button>
+);
+
+const suspendButton = (
+  <Button variant="outlined" size="small">
+    Suspend
+  </Button>
+);
+
+const acceptButton = (
+  <Button
+    variant="contained"
+    color="primary"
+    size="small"
+    startIcon={<CheckCircleIcon />}
+    disableElevation
+  >
+    Accept
+  </Button>
+);
+
+const viewButton = (
+  <Button
+    variant="contained"
+    color="primary"
+    size="small"
+    startIcon={<InfoIcon />}
+    disableElevation
+  >
+    View
+  </Button>
+);
+
+const resetButton = (
+  <Button
+    variant="contained"
+    color="primary"
+    size="small"
+    startIcon={<InfoIcon />}
+    disableElevation
+  >
+    Reset Password
+  </Button>
+);
+
+type Props = {
+  data: Array<OrgApp | Organization | User>;
+  pageType: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function giveButton1(pT: string) {
+  if (pT === 'applications') {
+    return declineButton;
+  }
+  if (pT === 'organizations') {
+    return deleteButton;
+  }
+  return suspendButton;
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function giveButton2(pT: string) {
+  if (pT === 'applications') {
+    return acceptButton;
+  }
+  if (pT === 'organizations') {
+    return viewButton;
+  }
+  return resetButton;
+}
```

Comment:
move this inside the component body and use the return type `React.ReactElement`
you can also use arrow functions like
```typescript
const button = () => {...}
```
inside the function, you also don't have pass in an param bc you have access to pageType